### PR TITLE
Smoke: Separated scripts for SendLocation and SetAppIcon from smoke API script 

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Separated scripts for SendLocation and SetAppIcon from smoke API script

### CODE REVIEW

File: test_scripts/Smoke/API/018_SendLocation_PositiveCase.lua [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+--[[ Local Functions ]]
+local function ptuUpdateFuncRPC(tbl)
+  local SLgroup = {
+    rpcs = {
+      SendLocation = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" }
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = SLgroup
```
Comment (aderiabin) [inline test_scripts/Smoke/API/018_SendLocation_PositiveCase.lua]: @GetmanetsIrina I would propose to use "point" notation for constant string keys in tables and "braces" notation for variable string keys and other types keys.
Examples:
```
tbl.policy_table.functional_groupings.NewTestCaseGroup = SLgroup
```
but
```
local groupName = "NewTestCaseGroup"
tbl.policy_table.functional_groupings[groupName ] = SLgroup
```

File: test_scripts/Smoke/API/019_SetAppIcon_PositiveCase.lua [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  requestParams = requestParams,
+  requestUiParams = requestUiParams
+}
+
+--[[ Local Functions ]]
+local function setAppIcon(params, self)
+  local cid = self.mobileSession1:SendRPC("SetAppIcon", params.requestParams)
+  params.requestUiParams.appID = commonSmoke.getHMIAppId()
+  EXPECT_HMICALL("UI.SetAppIcon", params.requestUiParams)
+  :Do(function(_,data)
```
Comment (aderiabin) [inline test_scripts/Smoke/API/019_SetAppIcon_PositiveCase.lua]: @GetmanetsIrina Please add space between function parameters.
'function(_,data)' => `function(_, data)`

File: test_scripts/Smoke/API/018_SendLocation_PositiveCase.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/018_SendLocation_PositiveCase.lua]: @aderiabin , Done in 87403972486a44f8eb7fcf65dcbe04200182c50f

File: test_scripts/Smoke/API/019_SetAppIcon_PositiveCase.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/019_SetAppIcon_PositiveCase.lua]: @aderiabin , Done in 87403972486a44f8eb7fcf65dcbe04200182c50f

**Pull request merged**